projects/packages/my-jetpack/_inc/components/product-cards-section/boost-card/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+import React from 'react';
+import ProductCard from '../../connected-product-card';
+import { PRODUCT_STATUSES } from '../../product-card/action-button';
+import BoostSpeedScore from './boost-speed-score';
+
+const BoostCard = ( { admin }: { admin: boolean } ) => {
```

Comment:
```suggestion
const BoostCard = ( { admin: boolean } ) => {
```

I believe this can be refactored to just this, also I have mostly seen types include the FC or FunctionComponent type as well. [Here is an example](https://github.com/Automattic/jetpack/blob/trunk/projects/js-packages/components/components/dialog/index.tsx#L26), maybe we could do it like that here as well

Reply:
Yes, it can be done either way, like:
`const BoostCard = ( { admin }: { admin: boolean } ) => {`
Or:
`const BoostCard: React.FC< { admin: boolean } > = ( { admin } ) => {`

However, we cannot do it like your suggested change:
`const BoostCard = ( { admin: boolean } ) => {`
Because here we are actually aliasing (renaming) the `admin` prop to be `boolean`, (not the type, but actually re-naming it to a variable called boolean).

I'll go ahead and change it to use the `React.FC` type. 👍🙂

Reply:
You're right, I was thinking of the FC syntax 😆